db/migrate/20240722120510_add_claim_window_to_claims.rb

```diff
@@ -0,0 +1,5 @@
+class AddClaimWindowToClaims < ActiveRecord::Migration[7.1]
+  def change
+    safety_assured { add_reference :claims, :claim_window, foreign_key: true, type: :uuid }
```

Comment:
I don't think this should be wrapped in `safety_assured`.
`strong_migration` should suggest you include `algorithm: :concurrently`.

Reply:
It does suggest that, as it assumes there is existing table that will need to be locked in order to populate the index. As this is a relatively small table, I'd urge for the simplicity of this migration and go forward with the `safety_assured` route.

There is very low risk this will be problematic or long-running.

Reply:
I've just run it myself and it suggests:

```
=== Dangerous operation detected #strong_migrations ===

Adding a foreign key blocks writes on both tables. Instead, use:

class AddClaimWindowToClaims < ActiveRecord::Migration[7.1]
  disable_ddl_transaction!

  def change
    add_reference :claims, :claim_window, type: :uuid, index: {algorithm: :concurrently}
  end
end
```

I would recommend you do what strong migrations suggests, rather than wrap it in `safety_assured`

Reply:
This is now addressed.

```diff
  === Dangerous operation detected #strong_migrations ===
  
  Adding a foreign key blocks writes on both tables. Instead, use:
  
  class AddClaimWindowToClaims < ActiveRecord::Migration[7.1]
    disable_ddl_transaction!
  
    def change
      add_reference :claims, :claim_window, type: :uuid, index: {algorithm: :concurrently}
    end
  end

+ Then add the foreign key in separate migrations.
```

There is an additional step. I'll add the foreign key in a future PR. This is why I wanted to assume safety and skip this additional commit.